content/components/selectpanel.mdx

```diff
@@ -389,105 +376,7 @@ For multi-selections with search functionality, a convenient **deselect all** [i
   </Dont>
 </DoDontContainer>
 
-### Action menu trigger
-
-When the select panel is activated through an [action menu](/components/action-menu), it should be displayed as a modal dialog. It should never be shown as a submenu.
-
-<DoDontContainer>
-  <Do>
-    <img
-      width="464"
-      alt="An action menu is presented with multiple items. The first item, 'Visible to...', is currently hovered, and it is annotated with 'Open as modal dialog'."
-      src="https://github.com/primer/design/assets/980622/22386756-2612-40dd-aceb-1f765095cf17"
-    />
-    <Caption>When the select panel is triggered by an action menu, it should be presented as a modal dialog.</Caption>
-  </Do>
-  <Dont>
-    <img
-      width="464"
-      alt="A action menu with multiple items and where the first item 'Visible to' has a arrow on the right of the item and opened a multi select panel upon hover."
-      src="https://github.com/primer/design/assets/980622/a71c59d6-ca35-49ac-b082-e72aa4d1a8cd"
-    />
-    <Caption>Avoid displaying the select panel as a non-modal submenu within an action menu.</Caption>
-  </Dont>
-</DoDontContainer>
```

Comment:
Why are we removing this section? I have a feeling we'll get questions on what is possible if we don't include this.